repos/c2sm/packages/fdb/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fdb(SpackFdb):
+    """FDB (Fields DataBase) is a domain-specific object store developed at
+    ECMWF for storing, indexing and retrieving GRIB data."""
+
+    version("5.11.17",
+            sha256=
+            "375c6893c7c60f6fdd666d2abaccb2558667bd450100817c0e1072708ad5591e")
```

Comment:
```suggestion
            "375c6893c7c60f6fdd666d2abaccb2558667bd450100817c0e1072708ad5591e")
            
    depends_on("ecbuild@3.7:", type="build", when="@5.11.6:")
```

Reply:
With 5.11.23 and this, it's an exact backport of https://github.com/spack/spack/pull/40571. What's your opinion on that?

Reply:
Yeah exactly, I would just like to have the later versions available with c2sm spack in the interim because when I demo FDB for example I suggest to install a spack environment with spack_c2sm containing 5.11. Is it a problem to backport temporarily? Just because I dont know when the next spack release will be.